src/H3Bites.js

```diff
@@ -19,49 +21,49 @@ const H3Bites = props => {
     return 0;
   }
 
-  return Object.entries(groupBy(h3SoundBites(), "speaker"))
-    .sort((a, b) => compareAttributes(a, b, 0))
-    .map(([groupName, bitesData]) => {
-      const heading = <h2 style={{width: 'fit-content'}}>{groupName}</h2>;
-      const image = <img src={bitesData[0].imgSrc} style={{width: '10vw', height: 'auto'}}/>;
+  function dataToSearch(data) {
+    return [data.speaker.toLowerCase()].concat(
+      data.name.split(" ").map(d => d.toLowerCase())
+    );
+  }
 
-      const header = (
-        <div
-          style={{
-            alignItems: "flex-start",
-            display: "flex",
-            flexDirection: "row-reverse",
-            justifyContent: "flex-end",
-          }}
-        >
-          {heading}
-          {image}
-        </div>
-      );
+  function speakerNameMatchesSearchTerms(name) {
+    let nameMatches = searchTerms.every(st => name.toLowerCase().includes(st));
+    if (nameMatches) return true;
 
-      const bites = bitesData.sort((a, b) => compareAttributes(a, b, "name")).map((bite, i) => {
-        return (
-          <Bite
-            key={i}
-            name={bite.name}
-            src={bite.src}
-            speaker={bite.speaker}
-          />
-        );
+    return false;
+  }
+
+  function foundMatch(matchData) {
+    if (speakerNameMatchesSearchTerms(matchData[0])) return true;
```

Comment:
Can't we have the return of _speakerNameMatchesSearch_ here like we have on line 39?

Reply:
Oh yes, duh!